daemon/container_operations_unix.go

```diff
@@ -84,7 +84,7 @@ func (daemon *Daemon) getPidContainer(container *container.Container) (*containe
 	containerID := container.HostConfig.PidMode.Container()
 	container, err := daemon.GetContainer(containerID)
```

Comment:
Wondering if we should change this variable's name back to just `c`, as I think it now masks the `container` argument (this was also changed in https://github.com/docker/docker/commit/12485d62eeba27119260dc5eb54ac4fa83c3130b)

There's two containers in this function: the "main" container, and the container which namespace it's joining.

(I see that commit changed the variable name in a couple of other functions as well)

Reply:
Yep, We should change this variable's name to just `c`. But I think we should open an new PR to do this. : )

Reply:
Agreed!